omod-1.8/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_8/PatientIdentifierTypeController1_8Test.java

```diff
@@ -130,6 +134,25 @@ public void shouldRetireAPatientIdentifierType() throws Exception {
 		assertEquals(reason, service.getPatientIdentifierTypeByUuid(getUuid()).getRetireReason());
 	}
 	
+	@Test
+	public void shouldUnRetireAPatientIdentifierType() throws Exception {
+		PatientIdentifierType patientIdentifierType = service.getPatientIdentifierTypeByUuid(getUuid());
+		patientIdentifierType.setRetired(true);
+		patientIdentifierType.setRetiredBy(Context.getAuthenticatedUser());
+		patientIdentifierType.setDateRetired(new Date());
+		patientIdentifierType.setRetireReason("random reason");
+		service.savePatientIdentifierType(patientIdentifierType);
```

Comment:
What happens when the above line is commented out?
Am now asking this same question for the third time. Do you take time to review your changes?

Reply:
Apologies on this @dkayiwa , I think I have been following my not up-to date list of the changes and resources affected as I have made on this pull request. The next commit will consider everything is fine. Thank you very much.